exporter/datadogexporter/factory_test.go

```diff
@@ -123,67 +123,6 @@ func TestCreateDefaultConfig(t *testing.T) {
 	assert.NoError(t, componenttest.CheckConfigStruct(cfg))
 }
 
-// Test that the factory creates the default configuration
-func TestCreateDefaultConfigLogsAgent(t *testing.T) {
```

Comment:
Why is this test removed?

Reply:
We added this test when the logs agent feature flag created a different default config, but now the default configs are the same with or without the logs agent feature gate.